images/build/Dockerfiles/cli.Dockerfile

```diff
@@ -36,6 +37,13 @@ RUN apt-get update \
     && ln -s /opt/buildscriptgen/GenerateBuildScript /opt/oryx/oryx \
     && echo "cli" > /opt/oryx/.imagetype
 
+RUN tmpDir="/opt/tmp" \
+    && cp -f $tmpDir/images/build/benv.sh /opt/oryx/benv \
+    && chmod +x /opt/oryx/benv
+
 ENV ORYX_SDK_STORAGE_BASE_URL="https://oryx-cdn.microsoft.io"
 ENV ENABLE_DYNAMIC_INSTALL="true"
-ENV PATH="$PATH:/opt/oryx"
\ No newline at end of file
+ENV PATH="$PATH:/opt/oryx"
+ENV DYNAMIC_INSTALL_ROOT_DIR="/opt"
```

Comment:
setting this `DYNAMIC_INSTALL_ROOT_DIR` to be `/opt` means we don't use the default path `/tmp/oryx/platforms` as other images. I'm wondering if it'll cause similar issues like we see last week?

Reply:
This would be something worth test. I know we have a `TemporaryInstallationDirectoryRoot` pointing to `/tmp/oryx/platforms`

Reply:
I was following the pattern set by `azureFunctions.JamStack.Dockerfile` -- do you happen to know why this image would want to set the dynamic install root path to `/opt` instead of the path you listed above?